build/build.js

```diff
@@ -71,11 +71,11 @@ gulp.task('build-styles', function () {
     .pipe(sass()
       .on('error', sass.logError))
     .pipe(sourcemaps.write())
-    .pipe(rename(function(path) {
+    .pipe(rename(function (path) {
     	path.basename = 'bc_vpaid_vast_mo';
     }))
     .pipe(gulp.dest(destPath))
     .pipe(size({showFiles: true, title: '[Styles]'}));
 });
 
-module.exports = new BuildTaskDoc('build', 'This task builds the plugin', 4);
+module.exports = new BuildTaskDoc('build', 'This task runs karma tests and builds the plugin', 1);
```

Comment:
Same comment as above - let's list these operations in the order they happen (build, then tests).

Reply:
Done